scripts/sam/eval_combo.py

```diff
@@ -316,7 +316,7 @@ def mlp_only(mod, name):
         apply_fake_sparsity(predictor.model.image_encoder,
                             filter_fn=mlp_only)
 
-        predictor.model.image_encoder = quantize(predictor.model.image_encoder,
+        predictor.model.image_encoder = quantize_(predictor.model.image_encoder,
```

Comment:
actually this is making it unclear if the API is in place or not, same comment will apply to the whole repo

Reply:
oh it should be inplace, will update